lib/Frontend/Frontend.cpp

```diff
@@ -309,7 +327,8 @@ void CompilerInstance::performSema() {
   }
 
   ModuleDecl *importedHeaderModule = nullptr;
-  StringRef implicitHeaderPath = options.ImplicitObjCHeaderPath;
+  StringRef implicitHeaderPath =
+      Invocation.getClangImporterOptions().BridgingHeader;
```

Comment:
Yeah, sorry, but please find another way to do this that doesn't require mutating the options. That could be as simple as checking in the importer whether we've loaded a PCH and ignoring the argument to import bridging header in that case. Or asserting that the PCH came from that header.